### ISSUE TITLE
Allow re-compute headers on each grpc call

### ISSUE THREAD

**Issue by asinghvi-aurora** [fullstorydev/grpcui#307]
When running grpc requests using bearer authentication I need to provide the bearer token in the header in this manner:

```
grpcui -H "Authorization: Bearer $(cat /path/to/token)"  ...
```

In my environment the tokens are short lived for security and are automatically regenerated at `/path/to/token`. Currently the token expires in 30 minutes and I need restart the grpcui web server every 30 minutes to continue to use it, otherwise requests fail authentication. It would be useful if grpcui could optionally allow some headers to be recomputed on each request, so that the same instance of the server can be used for a while.

**Comment by dragonsinth** [comment]
That `$(cat /path/to/token)` happens in bash, before grpcui is ever invoked...

**Comment by asinghvi-aurora** [comment]
Yes that's true, so I guess what I'm asking for here is to be able to run a command before each request where-in I can update a environment variable which I can use with `--expand-headers` to populate the token. Maybe there is a better way?
